src/components/IGVFileSelector.js

```diff
@@ -83,41 +123,69 @@ export const getValidIgvFiles = (values) => {
       }
 
       // Filter to URLs that point to a file with one of the relevant extensions.
-      const basename = url.pathname.split('/').at(-1);
-      const [base, extension] = splitExtension(basename);
-      return !!base && allFiles.includes(extension);
+      const filename = url.pathname.split('/').at(-1);
+      const isValid = hasValidIgvExtension(filename);
+      return isValid;
```

Comment:
```suggestion
      return hasValidIgvExtension(filename);
```

Reply:
Done in [7852932](https://github.com/DataBiosphere/terra-ui/pull/5191/commits/7852932915f0079c65f94b0b97eed9beb98f1407).